spec/system/admin/widgets/cards_spec.rb

```diff
@@ -25,6 +26,7 @@
         expect(page).to have_content "Card description"
         expect(page).to have_content "Link text"
         expect(page).to have_content "consul.dev"
+        expect(page).to have_css("td", exact_text: "12")
```

Comment:
Since none of the methods above or below this one use parentheses, do you think it makes sense to remove them here too? :thinking:

Same thing in a few other changes in this pull request where we could `have_css "` instead of `have_css("`.

```suggestion
        expect(page).to have_css "td", exact_text: "12"
```